Frameworks/include/CGPathInternal.h

```diff
@@ -56,5 +57,6 @@ struct CGPathElementInternal : CGPathElement {
 typedef struct CGPathElementInternal CGPathElementInternal;
 
 HRESULT _CGPathGetGeometry(CGPathRef path, ID2D1Geometry** pGeometry);
+HRESULT _CGPathApplyInternal(Microsoft::WRL::ComPtr<ID2D1PathGeometry> pathGeometry, void* info, CGPathApplierFunction function);
 
```

Comment:
`_CGPathApplyInternal(ID2D1PathGeometry* pathGeometry)`

---
In reply to: [91632808](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r91632808) [](ancestors = 91632808)